### PR TITLE
Code improvements

### PR DESCRIPTION
**Pull request by HackoDev**
* `type_` renamed to `category`
* `__all__` added
* `print_expenses` for-loop simplified
* `test_expenses` definition list updated

### CODE REVIEW

File: refactor_me.py [hunk 1]
```diff
@@ -1,26 +1,35 @@
 #!/usr/bin/env python3
 
+__all__ = ['Expense', 'sum_expenses', 'print_expenses']
```
Comment (dmugtasimov) [inline refactor_me.py]: can we use tuple here?

File: refactor_me.py [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
 
 if __name__ == '__main__':
     # TODO(dmu) HIGH: Use static fixtures and dynamic fixture framework instead
-    test_expenses = (Expense('food', 4), Expense('food', 3), Expense('car', 3), Expense('dog', 1))
-    print_expenses(sum_expenses(test_expenses, 2))
+    test_expenses = (
+        Expense('food', 4),
+        Expense('food', 3),
+        Expense('car', 3),
+        Expense('dog', 1)
```
Comment (dmugtasimov) [inline refactor_me.py]: well, then we need comma here to be consistent with your intention

File: refactor_me.py [hunk 3]
```diff
@@ -1,26 +1,35 @@
...
+    )
+    sum_results = sum_expenses(test_expenses, min_amount=2)
```
Comment (dmugtasimov) [inline refactor_me.py:35]: and extra line above

File: refactor_me.py (hunk 1 above)
Comment (HackoDev) [reply refactor_me.py]: sure, I will. i think `tuple` would be more preferable here

File: refactor_me.py (hunk 2 above)
Comment (HackoDev) [reply refactor_me.py]: agree, I missed that

**Pull request closed without merging**